### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by typwrtr**

### CODE REVIEW

File: conventions/README.md [hunk 1]
```diff
@@ -14,6 +32,100 @@ This directory contains *approved* conventions and represents best practices in
...
+  "$schema": "https://conventions.0xcert.org/86-base-asset-schema.json",
+  "description": "A weapon for the Troopers game which can severely injure the enemy.",
+  "image": "https://troopersgame.com/dog.jpg",
+  "name": "Magic Sword"
+}
+```
+
+## Rationale
+
+- Why micro-credentials
```
Comment (MoMannn) [inline conventions/README.md]: Same as comment before.

File: conventions/README.md [hunk 2]
```diff
@@ -14,6 +32,100 @@ This directory contains *approved* conventions and represents best practices in
...
+- What are they solving
+- Typical terms
+
+## Backwards Compatibility
+
+Currently, there is no schema defined for backwards compatibility, so it's backwards-related to everything blockchain. Blockchain-wise it has no problem. As for the current academic status, please elaborate on specific standards for academic credentials in Europe, and how this convention is compatible with existing EU standards for micro-credentials.
+
+## Implementations
+
+Use of ERC-721 non-fungible tokens (NFTs)
```
Comment (MoMannn) [inline conventions/README.md]: This are not actual implementation of conventions...

File: conventions/README.md [hunk 3]
```diff
@@ -2,10 +2,28 @@
...
+
+Every digital asset in the 0xcert Protocol is defined using a *convention*. Conventions are data models that allow your application to interoperate with other applications using the 0xcert Protocol.
+
+This convention is the base specification to which all other conventions MUST conform.
+
+This convention is minimal and adopts only the specification for digital assets from [ERC-721](https://eips.ethereum.org/EIPS/eip-721) while adding the requirement that every digital asset specify to which convention it conforms.
+
+## Motivation
+
+The European Qualification Framework (EQF) enables learners, learning providers and employers to compare qualifications between different national systems. The metadata is, therefore, an attempt to create a new or additional EQF schema within the EQF pillar in ESCO that proposes a set of standard metadata for documenting micro-credentials, and specifically on how to record, store and transfer them via computer systems.
```
[the remark this answers is not in the document]
Comment (typwrtr) [reply conventions/README.md]: These unintentional changes were removed.

File: conventions/README.md (hunk 1 above)
Comment (typwrtr) [reply conventions/README.md]: These unintentional changes were removed.

File: conventions/README.md (hunk 2 above)
Comment (typwrtr) [reply conventions/README.md]: These unintentional changes were removed.

**Pull request merged**